testTruckRender.gd

```diff
@@ -0,0 +1,56 @@
+@tool
+extends Sprite2D
+
+var truckModel;
+var postProcObject;
+var lastPos: Vector2;
+var currentPos: Vector2;
+var extend_vec: Vector2;
+
+var angle_to = 0;
+
+var lastTick = 0
+
+@export var modelTint: Color = Color.RED:
+	set(val):
+		modelTint = val;
+@export_range(0, 1, 0.1) var modelTintFactor:
+	set(val):
+		modelTintFactor = val;
+		
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	truckModel = $SubViewport/testTruck
+	postProcObject = get_node("SubViewport/Camera3D/Postprocess")
```

Comment:
Is there a reason for using $ and `get_node`? Are these different?

Reply:
I think this is just different ways of accessing the nodes. I should check to see what the actual difference is, I don't believe it's different from the get_node call though. 
I'd stolen some code that used the $ accessor, but I'd like to keep the style consistent so we might just do it all with the get_node calls